From 23b99a1fc0d05a801271aca218fbfbba9bb589ac Mon Sep 17 00:00:00 2001
From: Scott Tsai <scottt.tw@gmail.com>
Date: Sat, 22 Mar 2025 06:11:03 +0800
Subject: [PATCH] ck.hpp: define CK_BUFFER_RESOURCE_3RD_DWORD for gfx1151

Define CK_BUFFER_RESOURCE_3RD_DWORD with the same value for gfx110x and
gfx115x (0x31004000) following https://github.com/ROCm/hipBLASLt/pull/1766
---
 include/ck/ck.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/ck/ck.hpp b/include/ck/ck.hpp
index 1ec0c6bc2..5e096399c 100644
--- a/include/ck/ck.hpp
+++ b/include/ck/ck.hpp
@@ -70,7 +70,7 @@ CK_DECLARE_ENV_VAR_BOOL(CK_LOGGING)
 #define __gfx103__
 #endif
 #if defined(__gfx1100__) || defined(__gfx1101__) || defined(__gfx1102__) || \
-    defined(__gfx1103__) || defined(__gfx11_generic__)
+    defined(__gfx1103__) || defined(__gfx1150__) || defined(__gfx1151__) || defined(__gfx11_generic__)
 #define __gfx11__
 #endif
 #if defined(__gfx1200__) || defined(__gfx1201__) || defined(__gfx12_generic__)
-- 
2.47.1.windows.2

